eng/Directory.Build.Data.targets

```diff
@@ -10,15 +10,19 @@
     <Error Condition="'@(MissingTargetFrameworks)' != ''" Text="Missing required target frameworks '@(MissingTargetFrameworks)'. Please ensure you add those frameworks." />
   </Target>
 
-  <!-- Set PackageProjectUrl to the package README for Client Libraries -->
+  <!-- Set PackageProjectUrl and PackageReleaseNotes to the package README.md and CHANGELOG.md respectively for Client Libraries -->
   <Target Name="SetPackageProjectUrl" BeforeTargets="ValidateTargetFrameworks" Condition="'$(IsClientLibrary)' == 'true' and '$(IsTestProject)' != 'true' and
    '$(IsSamplesProject)' != 'true' and '$(IsTestSupportProject)' != 'true' and '$(CommitSHA)' != ''">
     <PropertyGroup>
       <CurrentCsprojPath>$(MSBuildProjectDirectory)</CurrentCsprojPath>
       <DirectoryPartofPath>$(CurrentCsprojPath.Substring($(CurrentCsprojPath.LastIndexOf("\sdk\"))))</DirectoryPartofPath>
       <DirectoryPartofPath>$(DirectoryPartofPath.Remove($(DirectoryPartofPath.LastIndexOf("\src"))))</DirectoryPartofPath>
       <PackageProjectUrl>$(RepositoryUrl)/blob/$(CommitSHA)$(DirectoryPartofPath)/README.md</PackageProjectUrl>
+      <PackageReleaseNotes>$(RepositoryUrl)/blob/$(CommitSHA)$(DirectoryPartofPath)/CHANGELOG.md<PackageReleaseNotes>
     </PropertyGroup>
+    <ItemGroup>
+      <None Include="$(RepoRoot)/$(DirectoryPartofPath)/CHANGELOG.md" Pack="true" PackagePath="changelog.md"/>
```

Comment:
We should probably add some exists checks here and only include the file and the link if the file exists.

Reply:
Also, the file should be named CHANGELOG.md in the package. The standard convention is all uppercase sans file extension, so should be stored in teh package as such.